manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com//opendatahub-io/notebooks/tree/main/habana"
+    opendatahub.io/notebook-image-name: "HabanaAI"
+    opendatahub.io/notebook-image-desc: "Jupyter notebook image with a set of habana libraries that advanced AI/ML notebooks will use as a base image to provide a standard for libraries avialable in all notebooks"
+    opendatahub.io/notebook-image-order: "7"
+    opendatahub.io/recommended-accelerators: '["habana.ai/gaudi"]'
+  name: jupyter-habana-notebook
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  # 1.11.0 Version of the image n
+  - annotations:
+      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"},{"name":"Habana","version":"1.11.0.587"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.23"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Tensorflow","version":"2.12.1"},{"name":"PyTorch","version":"2.0.1"},{"name":"Elyra","version":"3.15"}]'
+      openshift.io/imported-from: quay.io/opendatahub/workbench-images
+    from:
+      kind: DockerImage
+      name: $(odh-habana-notebook-image-n)
+    name: "1.11.0"
+    referencePolicy:
+      type: Source
+  # 1.10.0 Version of the image n-1
+  - annotations:
+      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"},{"name":"Habana","version":"1.10.0.494"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.23"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Tensorflow","version":"2.12.0"},{"name":"PyTorch","version":"2.0.1"},{"name":"Elyra","version":"3.15"}]'
+      openshift.io/imported-from: quay.io/opendatahub/notebooks
+      opendatahub.io/workbench-image-recommended: 'true'
```

Comment:
Are we sure that we want 1.10 as recommended and note the 1.11?

Reply:
Initial thought was with 1.10, as that the operator version available on 4.12 openshift
however i have completely removed the recommendation for now.
as this would totally depend on the user.